components/zenler/actions/unenroll-user/unenroll-user.mjs

```diff
@@ -0,0 +1,48 @@
+import zenler from "../../zenler.app.mjs";
+
+export default {
+  key: "zenler-unenroll-user",
+  name: "Unenroll User",
+  description: "Unenrolls a user from a course. [See the docs here](https://www.newzenler.com/api/documentation/public/api-doc.html#faaadc39-5702-bb2c-640f-c76a47c26f82)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    zenler,
+    userId: {
+      propDefinition: [
+        zenler,
+        "userId",
+      ],
+    },
+    courseId: {
+      propDefinition: [
+        zenler,
+        "courseId",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const {
+      userId,
+      courseId,
+    } = this;
+
+    const response = await this.zenler.unenrollUser({
+      userId,
+      data: {
+        course_id: courseId,
+      },
+    });
+
+    if (typeof(response) === "string") {
+      console.log(response);
+      throw new Error("Response error");
```

Comment:
Is it possible to specify the error in the `throw` message?

Reply:
See [here](https://github.com/PipedreamHQ/pipedream/pull/4312#discussion_r981511609)